content/docs/intro/languages/dotnet.md

```diff
@@ -64,17 +64,14 @@ The fastest way to get up and running is to choose from one of the following Get
     <a class="btn bg-transparent border w-1/2 border-solid border-gray-300 hover:bg-gray-200 justify-center p-4 mr-4" href="{{< relref "../../get-started/gcp" >}}">
         <img class="h-5 ml-auto mr-auto" src="/logos/tech/gcp.svg" alt="Google Cloud">
     </a>
-    <a class="btn bg-transparent border w-1/2 border-solid border-gray-300 hover:bg-gray-200 justify-center p-4" href="{{< relref "../../get-started/kubernetes" >}}">
-        <img class="h-5 ml-auto mr-auto" src="/logos/tech/k8s.svg" alt="Kubernetes">
-    </a>
```

Comment:
We don't yet have .NET in the Kubernetes Get Started guide, so removing this temporarily for now.

Reply:
Ah 😢

Reply:
By the way, this table looked pretty bad on mobile when I looked this morning (wraps funnily) — this reaches the limits of my CSS skills... any idea why?

Reply:
> By the way, this table looked pretty bad on mobile when I looked this morning

Ah yes. I'll add a commit here that fixes it.